src/core/PlayArea.ttslua

```diff
@@ -47,13 +47,10 @@ local PLAY_AREA_ZONE_GUID = "a2f932"
 
 local clueData = {}
 local spawnedLocationGUIDs = {}
-
-local locations = { }
-local locationConnections = { }
-local draggingGuids = { }
-
+local locations = {}
+local locationConnections = {}
+local draggingGuids = {}
```

Comment:
This is not changed because of the spaces between the brackets, but because there were multiple empty lines.